### ISSUE TITLE
Comma in filenames

### ISSUE THREAD

**Issue by jawsdark** [sol/hpack#342]
Cabal needs path/filenames with commas to be enclosed in double quotes.
```
data-files:
  "path/some,file"
```
If the quotes are missing, then Cabal reads it as two different files:
```
data-files:
  path/some,
  file
```
If I use glob patterns like `path/*` the generated file contains `path/some,file` without quotes and the problem occurs. I did not find any way to enforce double quotes on path/filenames in the generated Cabal file. This could be solved by just putting double quotes around all files listed in the generated Cabal file.

**Comment by sol** [comment]
Hey! Thanks for reporting this.  As a workaround you probably need to specify those files by hand for now.

To fix this proper I think we should implement Cabal's escaping rules properly.  The `Cabal` library may already provide something that we can use.  PR welcome!

**Comment by googleson78** [comment]
This is now fixed in 89ecd99.
